asn1/LedgerLedgerLotocol.asn

```diff
@@ -107,18 +107,27 @@ CALL ::= CLASS {
 CallSet CALL ::= {
     -- All requests MUST use odd typeIds
     -- All responses MUST use even typeIds
-    {1 PacketRequest} |
-    {2 PacketResponse} |
-    {3 PrepareRequest} |
-    {4 PrepareResponse} |
-    {5 FulfillRequest} |
-    {6 FulfillResponse} |
-    {7 RejectRequest} |
-    {8 RejectResponse} |
-    {9 TransferStateRequest} |
-    {10 TransferStateResponse} |
-    {11 CustomRequest} |
-    {12 CustomResponse}
+    -- {1 reserved because this is the typeId of InterledgerProtocolPaymentPacket in
+    -- https://github.com/interledger/rfcs/edit/mj-lll-packet-types/asn1/GenericPacket.asn } |
+    {2 QuoteLiquidityRequest} |
+    {3 QuoteLiquidityResponse} |
+    {4 QuoteBySourceAmountRequest} |
+    {5 QuoteBySourceAmountResponse} |
+    {6 QuoteByDestinationAmountRequest} |
+    {7 QuoteByDestinationAmountResponse} |
+    {8 InterledgerProtocolError} |
+    -- skipping numbering to allow more future ILP-layer packet types
+    {254 CustomRequest} |
+    {255 CustomResponse} |
```

Comment:
I thought this would fit better inside the low number space than outside it, since it's "custom" and could be anything, whether Interledger-layer, or ledger-layer, or side protocol. We could even make two identical but separate CustomRequest/Response pairs, one for use on the Interledger layer and one for use on the ledger layer, but that seems like overkill.